src/test/java/com/bringframework/factory/AutowiredBeanPostProcessorTest.java

```diff
@@ -0,0 +1,75 @@
+package com.bringframework.factory;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.bringframework.registry.BeanDefinition;
+import com.bringframework.registry.BeanDefinitionRegistry;
+import com.bringframework.resolver.DependencyResolver;
+import java.lang.reflect.Field;
+import java.util.Collections;
+import java.util.Map;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class AutowiredBeanPostProcessorTest {
+
+  @Mock
+  DependencyResolver dependencyResolver;
```

Comment:
Please add here private modifier and in other lines where it is needed

Reply:
done